interpreter/interpreter_test.go

```diff
@@ -278,6 +278,11 @@ var (
 			},
 			out: types.False,
 		},
+		{
+			name: "in_empty_list",
+			expr: `6 in []`,
+			out:  types.False,
+		},
```

Comment:
This test seems unrelated to this PR - maybe move to another PR?

Reply:
Ah, sure thing. It must have gotten picked up when I was looking over coverage.